### ISSUE TITLE
fruitless clone obsoletions

### ISSUE THREAD

**Issue by Clare72** [FlyBase/drosophila-anatomy-developmental-ontology#1712]
We do not have terms for clusters of other neurons (e.g. dopaminergic clusters, dsx clusters...) and it seems inconsistent to have cluster terms for fru. It may also be the case that relationships to lineages are not even 1:1.

This may require substantial retrofits to curation (at VFB and FlyBase).

**Comment by Clare72** [comment]
Since there does not seem to be any expression curation using a 'restricted' qualifier with any of these, I think we could retrofit by replacing with the appropriate 'neuron' terms.
```
SELECT * 
-- get expression ids linked to fru lineage clones 
FROM cvterm c 
JOIN expression_cvterm ec 
	ON c.cvterm_id = ec.cvterm_id AND c.name LIKE 'adult fruitless%lineage clone' 
JOIN EXPRESSION e 
	ON ec.expression_id = e.expression_id 
-- check if any of these ids linked to a 'restricted' qualifier 
JOIN expression_cvterm ec2 
	ON e.expression_id = ec2.expression_id 
JOIN cvterm c2 
	ON c2.cvterm_id = ec2.cvterm_id AND c2.name ='restricted'
;
```
(no results)

**Comment by admclachlan** [comment]
For expression curation it seems I only ever need the fruitless neuron terms. Aren't these useful for Cachero et al., 2010 though, if lineage<>neuron is not 1:1?

**Comment by Clare72** [comment]
Yeah, we have some images annotated, which I would need to retrofit to use the neuron term.

My main issue is that we don't make terms for groups of other neurons, e.g. if we had an image of all the DM4-derived dsx pC1 neurons, we don't have a 'clone' term for this (and I don't want to maintain clone terms for all possible clones by gene expression!) so we would just use the 'neuron' term.

Another issue is that I think a 'neuroblast lineage clone' should probably be a (more or less) complete set of cells of a given lineage (though defs do not currently specify this), otherwise what is the point in these terms, we could just always use the 'neuron' term. The fruitless clones are not full lineage clones, so they should probably be a 'partial neuroblast lineage clone' or something (but I don't really want to make this!).

**Comment by admclachlan** [comment]
Makes sense, I have no issue with these obsoletions.
